packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -22,25 +23,33 @@ import { __ } from '@wordpress/i18n';
 import { useSelect } from '@wordpress/data';
 
 const SettingsSidebar = () => {
-	const sidebarName = useSelect(
-		( select ) =>
-			select( 'core/interface' ).getActiveComplementaryArea(
-				'core/edit-post'
-			),
-		[]
-	);
-	if (
-		! [ 'edit-post/document', 'edit-post/block' ].includes( sidebarName )
-	) {
-		return null;
-	}
+	const { sidebarName, keyboardShortcut } = useSelect( ( select ) => {
+		let sidebar = select( 'core/interface' ).getActiveComplementaryArea(
+			'core/edit-post'
+		);
+		if (
+			! [ 'edit-post/document', 'edit-post/block' ].includes( sidebar )
+		) {
+			if ( select( 'core/block-editor' ).getBlockSelectionStart() ) {
+				sidebar = 'edit-post/block';
+			}
+			sidebar = 'edit-post/document';
+		}
```

Comment:
This deserves a comment. Why do we now have a fallback instead of returning null?